File: video-series-mono/src/main/java/org/jesperancinha/video/event/VideoSeriesEvent.java
package org.jesperancinha.video.event;

import lombok.Data;
import lombok.Value;
import org.jesperancinha.video.data.Genre;

import java.math.BigDecimal;

@Value
@Data
public class VideoSeriesEvent {
    Long id;
    String name;
    Integer volumes;
    BigDecimal cashValue;
    Genre genre;
}

File: video-series-mono/src/main/kotlin/org/jesperancinha/video/event/VideoSeriesEvent.kt
package org.jesperancinha.video.event

import org.jesperancinha.video.data.Genre
import java.math.BigDecimal

data class VideoSeriesEvent(
    val id: Long? = null,
    val name: String? = null,
    val volumes: Int? = null,
    val cashValue: BigDecimal? = null,
    val genre: Genre? = null
)